app/data/projects.ts

```diff
@@ -11,36 +11,81 @@ export interface IProjects {
 
 export const projects: IProjects = {
     "1": {
-        name: "Project 1",
-        photo_url: "/publicis_square.png",
-        title: "Project title 1",
-        idea: "IDEA",
-        insight: "INSIGHT",
+        name: "Paw Patrol",
+        photo_url: "/paw.png",
+        title: "Paw Patrol",
+        idea: "One show is all it takes to make you cool",
+        insight: "87% of today's parents believe that having a friendship with their children is important",
         images: [
-            "/publicis_square.png",
-            "/publicis_square.png",
-            "/publicis_square.png",
+            "/paw cover.png",
+            "/paw bill.png",
+            "/paw social.png",
+            "/paw x indigo.png",
+            "/paw web.png",
+            "/paw book.png",
```

Comment:
remove space in name add `-` instead of space and add `/projects/` before each url